<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Port</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBaudRate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPort">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisconnect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtCommands">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>471</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefreshPorts">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh Ports</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPort">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBaudRate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLedOn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Led On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLedOff">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Led Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetAll">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset All</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Led Blink</string>
    </property>
    <widget class="QComboBox" name="cbLedBlinkRate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSendBlinkParameter">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Parameter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLedBlink">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Led Blink</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Led PWM</string>
    </property>
    <widget class="QComboBox" name="cbLedPWM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLedPWM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Led PWM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSendPWMParameter">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Parameter</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
